on_startup.py

```diff
@@ -42,69 +74,29 @@ def populate_higlass_data_directory(data_dir):
 
 def ingest_tilesets(data_dir):
     """
-   Ingest previously downloaded files into higlass-server w/ django
-   management command
-   :param data_dir: <String> Path to directory populated with data to ingest
-   """
-    files_to_ingest = glob.glob(
-        '{}*multires.*'.format(data_dir)
-    )
-
-    for filename in files_to_ingest:
+    Ingest previously downloaded files into higlass-server w/ django
+    management command
+    :param data_dir: <String> Path to directory populated with data to ingest
+    """
+    for refinery_node_uuid in TILE_SETS.keys():
+        refinery_node_info = TILE_SETS[refinery_node_uuid]
+        file_path = data_dir + refinery_node_info["file_name"]
```

Comment:
This comes goes into `filename`: Double check? changing kwarg would be out of line?